Makefile.inc

```
@@ -23,6 +23,8 @@ INCLUDES += \
 	-I$(mkfile_path)modules\
 	-I$(mkfile_path)modules/API\
 	-I$(mkfile_path)modules/calculators\
+	-I$(mkfile_path)modules/CameraCapture\
+	-I$(mkfile_path)../../common/library/libcpp-spike/include \
 	-I$(mkfile_path)modules/common\
 	-I$(mkfile_path)modules/motions\
 	-I$(mkfile_path)modules/CameraCapture \
```

-I$(mkfile_path)modules/CameraCapture\
	-I$(mkfile_path)../../common/library/libcpp-spike/include \
	これが２重になっている
```suggestion
	-I$(mkfile_path)modules\
	-I$(mkfile_path)modules/API\
	-I$(mkfile_path)modules/calculators\
	-I$(mkfile_path)modules/common\
	-I$(mkfile_path)modules/motions\
	-I$(mkfile_path)modules/CameraCapture \
```